specs/Alert_Notifications.adoc

```diff
@@ -0,0 +1,189 @@
+
+
+= Alert Notifications
+
+Alerting system is essential to spot the problem and notify it to the corresponding user.
+The thresholds for alerts will be configured in grafana and the alerts will be raised by grafana.
+These alerts are captured by monitoring-integration module and notified with proper 
+details to respective user via the notifier.
+
+
+== Problem description
+
+The present alerting system monitors all the events and retrieves alert specific events.
+The alert specific events are processed and stored again in etcd under /alerting/alerts/
+as alerts recognized by tendrl. This leads to excess read and writes to etcd.
+Present implementation causes the alerting component to process events even
+when it can be done by a separate component before inserting the event in etcd.
+
+== Use Cases
+
+Goal: Notify user in case of any alert event.
+
+Actors:
+
+1) User
```

Comment:
In the context of this spec, the actor is the Notifier and not the user. The user is just a passive receiver, who is never directly involved.